src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2742,48 +2750,56 @@ msgstr "Organisation wurde erfolgreich erstellt"
 msgid "Organization was successfully saved"
 msgstr "Organisation wurde erfolgreich gespeichert"
 
-#: views/pages/page_actions.py:42
+#: views/pages/page_actions.py:45
 msgid "Page was successfully archived"
 msgstr "Seite wurde erfolgreich archiviert"
 
-#: views/pages/page_actions.py:65
+#: views/pages/page_actions.py:68
 msgid "Page was successfully restored"
 msgstr "Seite wurde erfolgreich wiederhergestellt"
 
-#: views/pages/page_actions.py:98
+#: views/pages/page_actions.py:101
 msgid "You cannot delete a page which has children"
 msgstr "Sie können keine Seite löschen, die Unterseiten besitzt"
 
-#: views/pages/page_actions.py:101
+#: views/pages/page_actions.py:104
 msgid "Page was successfully deleted"
 msgstr "Seite wurde erfolgreich gelöscht"
 
-#: views/pages/page_actions.py:229
+#: views/pages/page_actions.py:129
+msgid "URL '{}' was successfully copied to clipboard."
+msgstr "Die URL '{}' wurde erfolgreich in die Zwischenablage kopiert."
```

Comment:
Do we need the article "Die"? The English sentence does not start with "The" either. I removed the "Der" in front of "Ort" in the previous translation unification PR as well.

Reply:
You're right, let's go for full consistency here :muscle: